src/codegen/operator/table_scan_translator.cpp

```diff
@@ -105,6 +105,8 @@ class TableScanTranslator::ScanConsumer : public codegen::ScanCallback {
                               llvm::Value *tid_end,
                               Vector &selection_vector) const;
 
+  void RecordReads(CodeGen &codegen, Vector &selection_vector) const;
```

Comment:
There has been discussion of the transaction read-write set, recording writes and recording reads, that I was not immediately clear on what this function does. Should it be renamed to be "PerformReads" to more accurately describe what it does?

Reply:
That sounds reasonable.